README.md

```diff
@@ -20,6 +20,8 @@ Create the main logger instance :
 logger = KibanaLogger({"app": "your_app_name"})
 ```
 
+> You can configure the log level using the ENV variable `KIBANA_LOGGER_LOG_LEVEL` (DEBUG, INFO (default), WARNING, ERROR)
```

Comment:
it should be in a dedicated section rather than in a `quote`  section

Reply:
done